java/arcs/core/storage/DevToolsForStorage.kt

```diff
@@ -0,0 +1,30 @@
+package arcs.core.storage
+
+/**
+ * DevTools API exposed to the Storage Stack.
+ */
+interface DevToolsForStorage {
+
+  fun forDirectStore(options: StoreOptions): DevToolsForDirectStore
+  fun forRefModeStore(options: StoreOptions): DevToolsForRefModeStore
```

Comment:
KDoc please :)

Also, these methods seem to be named for the type of thing they return, rather than what they do. It may be clearer if they create new DevToolsForX instances to be called `createToolsForDirectStore`.. however, if they return a shared tools object, maybe `getToolsForDirectStore`.

Reply:
A few questions:

these implement methods from the interface, would you consider the documentation on the interface sufficient? I hesitate before copy pasting the same line of KDoc in multiple places.

These have been named to be readable as an entire expression, typically: `directStoreDevTools = devTools?.forDirectStore(options)`, which I consider shorter and nicer than more explicit: `devtools.getToolsForDirectStore(options)`.

Thoughts?

Reply:
Ah, these were on the interface. I thought I did document the interface, KDoc added now!

I will keep the naming as is you don't mind, the reading of `devTools.forDirectStore`was intentional.